### ISSUE TITLE
Upright Parentheses And Brackets in Italic Text

### ISSUE THREAD

**Issue by amarakon** [georgd/EB-Garamond#170]
I want to typeset all parentheses and brackets upright even in italic text. I can do this in LaTeX using the `embrac` package which automatically makes them upright. The downside of this approach is that I have to manually adjust the kerning to avoid collisions. I think it would be the most ideal if the italic font had a stylistic set that keeps parentheses and brackets upright with good kerning to take care of any potential collisions.

```tex
\documentclass{article}

\usepackage{fontspec}
\usepackage{embrac}

\setmainfont[ItalicFont = * 12 Italic]{EB Garamond}

\AddEmph{⸢}[0pt, 0pt]{⸣}[0.3em, 0pt]
\ChangeEmph{(}[0.2em, 0pt]{)}[0pt, 0pt]
\ChangeEmph{[}[0.2em, 0pt]{]}[0pt, 0pt]

\begin{document}

\emph{⸢f⸣}
\emph{(just)}
\emph{[just]}

\end{document}
```

![import-2023-04-04_18:35:18](https://user-images.githubusercontent.com/92645199/229937305-f3e19ba9-b806-4c46-9d8c-7be4ab788d5c.png)

**Comment by georgd** [comment]
What's the background of this usage? Is it a common requirement in a certain field?

**Comment by amarakon** [comment]
It is simply a typographic convention. *The Elements of Typographic Style* by Robert Bringhurst says:

> Parentheses and brackets are not letters, and it makes little sense to speak of them as
roman or italic. There are vertical parentheses and sloped ones, and the parentheses on italic fonts are almost always sloped, but vertical parentheses are generally to be preferred.
That means they must come from the roman font, and may need extra spacing when used
with italic letterforms

**Comment by RGB-es** [comment]
> It is simply a typographic convention. _The Elements of Typographic Style_ by Robert Bringhurst says:
> 

That depends on the language, it seems. In Spanish, if the enclosed text is all in italic the parenthesis _must_ be in italic, but if the enclosed text is mixed, then you must use the upright parenthesis.

**Comment by georgd** [comment]
Historically, at least, Granjon created sloped round parentheses for his Italics.

**Comment by timgrei** [comment]
I also read that as a recommendation in German. I think it was by Tschichold.

**Comment by georgd** [comment]
> I also read that as a recommendation in German. I think it was by Tschichold.

I've been waiting for this one 😉. Sounds very much like Bringhurst and Tschichold.

I think, I'll implement it but it'll have to wait until after unifying the two versions. Major changes won't happen in the 0.016 branch any more.

**Comment by amarakon** [comment]
Update: I think you should make this the default behavior instead of something that should have to be turned on by the user through an OpenType feature. Just like how the vertical bar (|) is not slanted in italic faces, it does not make sense for parentheses and brackets to be slanted in italic faces. Renaissance typographers used unslanted parentheses in italic text, even if they are part of the italic text and not just surrounding it. Perhaps there should be a feature to make them slanted, but that should not be the default behavior.

**Comment by georgd** [comment]
No, that's not true for all renaissance typefaces. The earlier Venetian italic typefaces usually had upright uppercase letters so it's very probable that no slanted parentheses were used. This can be seen

**Comment by georgd** [comment]
No, that's not true for all renaissance typefaces. The earlier Venetian italic typefaces usually had upright uppercase letters so it's very probable that no slanted parentheses were used. This can be seen in early French types too. But italics by messieurs Granjon and Garamont definitely do contain slanted parentheses.

This said, upright parentheses won't replace the slanted ones but I am committed to adding them as an alternative.

**Comment by georgd** [comment]
[Sorry for the editing/closing/reopening chaos. I'm currently on my phone and clumsily stumbled over the controls]
